vllm/config.py

```diff
@@ -836,6 +836,11 @@ class SchedulerConfig:
             swapping. However, when the sequence group has multiple sequences
             (e.g., beam search), recomputation is not currently supported. In
             such a case, we use swapping instead.
+        _send_delta_data: Private API. If used, scheduler sends delta data to
```

Comment:
dealta optimization should not be limited to ray. please make it a normal argument rather than marking it as private.